portfolio/src/main/webapp/script.js

```diff
@@ -27,9 +27,45 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function getMessage() {
-    const fetchPromise = fetch('/data');
-    fetchPromise.then(handleResponse);
+function getMessage(selectedObject) {
+    const language = selectedObject.value;
+    
+    if(language == "en") {
+        const fetchPromise = fetch('/data?language=en');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "es") {
+        const fetchPromise = fetch('/data?language=es');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "hi") {
+        const fetchPromise = fetch('/data?language=hi');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "pt") {
+        const fetchPromise = fetch('/data?language=pt');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "it") {
+        const fetchPromise = fetch('/data?language=it');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "fr") {
+        const fetchPromise = fetch('/data?language=fr');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "ht") {
+        const fetchPromise = fetch('/data?language=ht');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "he") {
+        const fetchPromise = fetch('/data?language=he');
+        fetchPromise.then(handleResponse);
+    }
+    else if(language == "ja") {
+        const fetchPromise = fetch('/data?language=ja');
+        fetchPromise.then(handleResponse);
+    }
```

Comment:
We can remove the `fetchPromise.then(handleResponse);` from each condition and just add it here at the end since this will be executed no matter the condition.